partner_email_check/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Follow the guideline: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst